src/sections/ProjectSection.js

```diff
@@ -34,4 +45,19 @@ const ProjectSection = ({ project, editing, onFormDataChange }) => {
   );
 };
 
+ProjectSection.fragments = {
```

Comment:
These are really nice - maybe at some point we should consider moving these into their own query file as the data is used throughout the application.

Reply:
I definitely think we can use fragments a lot better. I'm not sure how I feel about putting them in their own query files though as that seems to have less of the original benefit of graphql of co-locating queries next to components.